src/python/tools/configuration/marius_config.py

```diff
@@ -781,6 +787,13 @@ def type_safe_merge(base_config: MariusConfig, input_config: DictConfig):
 
     return base_config
 
+def create_model_output_path(params_output_dir):
+    params_output_path = Path(params_output_dir)
+    params_output_path.mkdir(parents=True, exist_ok=True)
+
+    for subdir in ["nodes", "edges"]:
```

Comment:
I think we don't need the `nodes` and `edges` directory anymore here. The model can just be stored like the following.

```
model_dir/
  model.pt
  model_state.py
  embeddings.bin
  embeddings_state.bin
  node_mapping.csv      // we should copy the mapping files to this directory too
  rel_mapping.csv  
  full_config.yaml      // would be good to have the configuration file that was used to train the model
```